config/config.txt

```diff
@@ -492,6 +492,9 @@ MINUTE_CLICK_LIMIT 400
 ## Send a message with the station name starting a new game.
 #CHAT_ANNOUNCE_NEW_GAME
 
+## Send a message with the round ID when a game has just ended.
```

Comment:
I don't love the wording on this, this makes it sound like a flag and not a string, especially with the name of the config.

Can you write a better description and potentially rename the config entry?

Reply:
I renamed both of them